rllib/agents/ppo/tests/test_ppo.py

```diff
@@ -164,7 +164,7 @@ def get_value():
             init_std = get_value()
             assert init_std == 0.0, init_std
 
-            if fw == "tf" or fw == "eager":
+            if fw != "torch":
```

Comment:
Let's try to be future proof here, `if fw in ["tf", "tfe"]:`

Reply:
+1

Reply:
done